graylog2-web-interface/src/components/lookup-tables/Cache.jsx

```diff
@@ -26,7 +27,12 @@ const Cache = React.createClass({
     return (
       <Row className="content">
         <Col md={6}>
-          <h2>{cache.title} <small>({plugin.displayName})</small></h2>
+          <h2>
+            {cache.title}
+            <ContentPackMarker contentPack={cache.content_pack} marginLeft={5} />
```

Comment:
After some discussion we've decided that we can postpone the creation of additional components for now.
It's really outside the scope of this feature and ideally we would restructure the content packs anyway to make them extensible.